code/datums/ruins.dm

```diff
@@ -0,0 +1,22 @@
+/datum/map_template/ruin
+	//name = "A Chest of Doubloons"
+	name = null
```

Comment:
It does mean that the name of the ruin and the filename are linked, and tbh, not all of the filenames are that accurate anymore. I'm planning on overhauling the template system anyway, and having proper names and descriptions for them would be very helpful.
"lavaland_cube" doesn't tell me as much as "Wishgranter Cube"